src/main/java/com/hivemq/client/internal/mqtt/MqttWebSocketConfigImplBuilder.java

```diff
@@ -61,8 +64,15 @@
         return self();
     }
 
+    public @NotNull B handshakeTimeout(final long timeout, final @Nullable TimeUnit timeUnit) {
```

Comment:
timeUnit is annotated Nullable but checked for not null.

Reply:
same here